Engines/Wine/Shortcuts/Reader/script.js

```diff
@@ -76,10 +73,10 @@ class ShortcutReader {
     }
 
     /**
-     * sets shortcut
-     * @param {string} shortcut shortcut
-     * @returns {void}
-     */
+	 * sets shortcut
```

Comment:
The indentation of these comments looks weird.

Reply:
It seems like I had some custom settings on my PC that were the reason for this. Should be fixed now.